torch/nn/_functions/rnn.py

```diff
@@ -342,11 +342,48 @@ def backward_extended(self, grad_output, grad_hy):
         return grad_input, grad_weight, grad_hx
 
 
-def RNN_symbolic_builder(*args, **kwargs):
-    def symbolic(g, input, all_weights, hx, **kwargs):
-        # Something can go here, e.g.
-        # return g.op('LSTM', input, *all_weights[0], outputs=2)
-        raise RuntimeError("RNN symbolic NYI")
+def RNN_symbolic_builder(cell_type, input_size, hidden_size, num_layers, batch_first, dropout, bidirectional, **kwargs):
+    assert cell_type == 'LSTM'
+    assert not batch_first
+    assert not dropout
+    assert not bidirectional
+
+    def symbolic(g, input, all_weights, h0_and_c0, **fkwargs):
+        h0, c0 = h0_and_c0
+        sequence_len = input.type().sizes()[0]
+        batch_size = input.type().sizes()[1]
+
+        # TODO leave out this argument to increase parametricity
```

Comment:
Can you create a task for this TODO?

Reply:
created an internal task to track, and expanded the comment with an explanation